torch/nn/quantized/modules/conv.py

```diff
@@ -52,15 +54,14 @@ def __init__(self, in_channels, out_channels, kernel_size, stride=1,
         qbias = torch._empty_affine_quantized([out_channels],
                                               scale=1, zero_point=0,
                                               dtype=torch.qint32)
-        self.register_buffer('_packed_weight',
-            torch.ops.quantized.fbgemm_conv_prepack(qweight, self.groups))
+        self.register_buffer('_packed_weight', torch.ops.quantized.fbgemm_conv_prepack(qweight, self.groups))
         self.register_buffer('bias', qbias)
         self.register_buffer('_scale', torch.tensor([1.0], dtype=torch.double))
```

Comment:
can you change the setter getter of scale, zero_point as well.

Reply:
You mean setters to explicitly set the tensor dtype? Will do!